stm32mp1.mk

```diff
@@ -116,10 +116,12 @@ OPTEE_OS_COMMON_FLAGS += \
 		CFG_RPMB_FS=y \
 		CFG_RPMB_TESTKEY=y \
 		CFG_REE_FS_ALLOW_RESET=y
-$(info -------------------------------------------------------------------------------------)
-$(info "WARNING: building with RPMB support for test purpose")
-$(info "         CFG_RPMB_FS_DEV_ID=$(CFG_RPMB_FS_DEV_ID) CFG_RPMB_FS=y, CFG_RPMB_TESTKEY=y and CFG_REE_FS_ALLOW_RESET=y")
-$(info -------------------------------------------------------------------------------------)
+$(info -------------------------------------------------------------------)
+$(info WARNING)
+$(info Building with RPMB support for test purpose. This enables OP-TEE)
+$(info unsafe RPMB test key CFG_RPMB_TESTKEY=y and request the RPMB device)
+$(info to be programmed accordingly.)
```

Comment:
It will not program the key since you do not set `CFG_RPMB_WRITE_KEY=y`.

Reply:
I don't want this build to force `CFG_RPMB_WRITE_KEY=y`. This must remain a decision of the caller. One does not want to realize too late that he/she has programmed the RPMB key without an explicit will.

Reply:
May I can add a setnence `To program the RPMB key, use CFG_RPMB_WRITE_KEY=y.`

I think the message should clarify that RPMB emulation is disabled.

Reply:
> I don't want this build to force `CFG_RPMB_WRITE_KEY=y`. This must remain a decision of the caller. One does not want to realize too late that he/she has programmed the RPMB key without an explicit will.

That was my point. "...and request the RPMB device to be programmed accordingly" sounds like the default config *will* program the key. Especially since there is a big WARNING. In fact I think no warning at all is needed, this is just a test config that cannot cause any permanent issue.

Reply:
Right. I guess I can fully remove the warning message, and just give clear explaination in the inline comment in _stm32mp1.mk_. I'll update accordingly.